sentry.client.config.ts

```diff
@@ -36,7 +36,7 @@ const lazyLoadSentryIntegrations = async () => {
   const { addIntegration, replayIntegration, browserTracingIntegration } =
     await import('@sentry/nextjs');
   addIntegration(replayIntegration({ maskAllText: false }));
-  addIntegration(browserTracingIntegration({ enableInp: true }));
+  addIntegration(browserTracingIntegration());
```

Comment:
no option in v8

Reply:
we haven't ported INP from v7 to v8 - see https://github.com/getsentry/sentry-javascript/issues/10945

but in v8 you won't need `enableInp` option, it will be enabled by default!